vcpkg/users/port-overlays.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Port overlays
+description: Port overlays
+author: data-queue
+ms.author: danshaw2
+ms.date: 9/20/2023
+ms.prod: vcpkg
+ms.topic: conceptual
+---
+
+# Port Overlays
+
+## Overview
+The default catalog in vcpkg comprises libraries maintained by both the vcpkg team and users. The catalog does not contain the entire universe of libraries, every possible version or patch.
+
+To facilitate the usage of these missing libraries without having to either add them to the default catalog or manage a custom registry, vcpkg uses port overlays. Port overlays are effective when you don't want to worry about versioning and simply want to use a library.
+
+## Overlay Ports
+An overlay port typically refers to a new port or a drop-in replacement for an existing port. In either case, the overlay port takes priority in vcpkg dependency resolution, ensuring it gets selected. However, a potential pitfall is the naming conflict that can arise when multiple overlays introduce the same port.
```

Comment:
```suggestion
An overlay port can act as a drop-in replacement for an existing port. An overlay port can also add a new port that is otherwise not available in a [registry](../maintainers/registries.md). While [resolving package names](registries.md#package-name-resolution), overlay ports take priority, forcing the overlay ports to be selected. 
```